### ISSUE TITLE
Симулятор

### ISSUE THREAD

**Issue by stason251187** [SBProX/SBProX-Dev#1282]
Здравствуйте!
Появились кнопки "Продолжить" и "Пауза" в окне при режиме Симулятор. Это хорошо!
Но! При нажатии одной из этих кнопок, меняется "длинна" панели. Т.е. длинна слова "продолжить" и "пауза" разная, и меняется. Т.е. если я нажму на кнопку "продолжить" примерно в области первых букв слова и не буду смещать курсор, и сразу же мне нужно поставить на "пауза", то на этом же месте находиться кнопка "Стоп". И тогда все труды заканчиваются! Уже 10 раз приходилось начинать заново.....  Исправьте, пожалуйста!

http://prntscr.com/szlwj6
http://prntscr.com/szlws5

**Comment by stason251187** [comment]
Странно! Написал один текст, а вижу в этом посте "сокращённый" вариант.... Незнаю видите ли Вы полный вариант текста данного поста???

**Comment by WebSurfer001** [comment]
Текст вижу весь. Скрины можно вставлять прямо в сообщение.

**Comment by SBProX** [comment]
Хорошо, что-то придумаем.

**Comment by dmo21749** [comment]
И пожалуйста, сделайте в симуляторе горячую клавишу на клавиатуре. Продолжить и пауза.
Или новый пост создать?

**Comment by AlTuzov** [comment]
Да, это было бы хорошо поправить, тоже уже несколько раз по ошибке нажимал на "стоп".
А вот кнопки +1TF, +2TF, +3TF, +D на мой взгляд особо и не нежны, на их место прекрасно бы встали "Старт\стоп" и "Пайза\продолжить" в фиксированного размера.

**Comment by Kesinzuri** [comment]
Поддерживаю - кнопку Стоп нужно защитить от случайного нажатия, либо горячими клавишами, либо запросом подтверждения на остановку симулятора. При наработке стратегии можно набирать статистику несколько дней и все данные потерять одним нажатием.

**Comment by blalorvet** [comment]
Поддерживаю! То что вынесли кнопку на панель это очень удобно, но то что меняется длинна плохо. Когда цена подходит к уровню то ты не смотришь на кнопку и не двигаешь мышку а просто нажимаешь и когда кнопка уменьшается при повторном нажатии ты закрываешь симулятор. Очень обидно.

**Comment by blalorvet** [comment]
Кстати кнопку стоп можно убрать с этой панели. Она нужна только  когда хочешь совсем закрыть тестирование, а перед закрытием обычно идешь и смотришь все результаты, сохраняешь их и тд.
И защищать от случайного нажатия тоже надо сделать, можно случайно стереть данные всего дня и не будет материала для анализа по своим сделкам, и целый день пропал зря.

**Comment by AlTuzov** [comment]
Согласен, кнопку стоп лучше вообще убрать с этой панели. Её нажимаешь только при старте симулятора и для завершения работы.

**Comment by dmo21749** [comment]
Я уже закалебался эту кнопку стоп по ошибке нажимать. И все приходится сначала начинать.
Уберите кнопку Стоп сверху, или сделайте горячие клавиши Стоп, Пауза, Продолжить.

**Comment by vomma-opt** [comment]
Кнопку Стоп нужно защитить от случайного нажатия, запросом подтверждения на остановку симулятора. предложение фиксированной длинны кнопки поддерживаю
